### DISCUSSION TITLE
Request for domain transfer assistance

### DISCUSSION THREAD

**Discussion by simcheolhwan** [vercel/community#6473] [Account Issue]
### Question

Dear Vercel Support,

I am reaching out regarding an issue with my domain access following an automatic account conversion. Initially, my account, under the Hobby plan, was used for managing several domains and projects. This account was recently converted into a team account named after me, causing all projects to migrate to this new setup. Unfortunately, this transition left my domains inaccessible under the new team account structure.

I attempted to resolve this by using the "Move" function for the domains within the original account's admin page. Although I received a success notification, the domains did not transfer to the team account. Given the circumstances, my primary request is assistance with transferring these domains correctly to ensure continued operation of my projects.

Additionally, while my ideal solution would involve reverting my account back to its original personal (Hobby) account configuration to avoid such complexities, I understand that facilitating the domain transfer may be the more immediate and feasible action. Therefore, I am prioritizing the domain transfer issue but would still appreciate any guidance or support in potentially reverting my account to its previous state.

Thank you for your support.

**Comment by amyegan** [comment]
Hey @simcheolhwan. I can help you locate your missing domains. Please let me know the names and I will look them up in our system.

**Reply by simcheolhwan** [reply]
personal user id: `XBv5h8NRTGhNPXJkeyeHlCKx`
team id: `team_a40QSe8t7OX5WWYJwbPlxfjH`

**Reply by amyegan** [reply]
I see 5 domains on your personal account. You should be able to see them at https://vercel.com/account/domains

You said you tried to [move the domains](https://vercel.com/docs/projects/domains/transfer-your-domain) to your new team and got a confirmation message but the domain didn't transfer. Would you be able to try moving one of them again and share some screenshots or a recording that I can show the engineering team? That will help us investigate what went wrong.

After that please confirm that you want all five domains moved to your new team, or let me know which ones to move, and I can help you get them moved over.

**Reply by simcheolhwan** [reply]
I solved the problem. The issue wasn't that the move was impossible; it was just that my team wasn't showing up in the move interface. By manually entering the team name, I was eventually able to complete the domain move.
